content/ja/examples/controllers/nginx-deployment.yaml

```diff
@@ -16,6 +16,6 @@ spec:
     spec:
       containers:
       - name: nginx
-        image: nginx:1.15.4
+        image: nginx:1.7.9
```

Comment:
It has to be changed too

Reply:
I fetched upstream and rebased commits to that master.
So this lines updated to modern nginx version.